packages/grafana-alerting/src/grafana/mocks/util.ts

```diff
@@ -0,0 +1,15 @@
+import { faker } from '@faker-js/faker';
```

Comment:
This is a quite sizable library. Are we going to use it more often in the codebase?

Reply:
It's only defined in the `devDependencies` so won't be bundled with the production bundle – is the size still a concern?